lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -357,24 +365,26 @@ def decrement_positions_on_lower_items(position=nil)
           end
 
           # This has the effect of moving all the higher items down one.
-          def increment_positions_on_higher_items
+          def increment_positions_on_higher_items(avoid_id = nil)
```

Comment:
Assignment Branch Condition size for increment_positions_on_higher_items is too high. [15.03/15]<br>Inconsistent indentation detected.